implementations/rust/ockam/ockam_command/src/progress_display.rs

```diff
@@ -0,0 +1,119 @@
+use crate::{fmt_log, CommandGlobalOpts, Terminal, TerminalStream};
```

Comment:
Nice job separating the notification send/recieve mechanism (which go in ockam_api), from the progress reporting ("UI") aspects (which go in ockam_command).